### ISSUE TITLE
Why is day of the week disabled?

### ISSUE THREAD

**Issue by liudr** [adafruit/RTClib#77]
The library makes no attempts to read or correctly set day of the week. There must be some reasons but what were the reasons?

**Comment by cvmanjoo** [comment]
Liudr,

Yes even i came across it. I believe its because DS3231 doesn't calculate the day of week it just increments it when day changes. So we programmatically calculate and set it to increment properly. 

Which is logically expensive.

**Comment by drak7** [comment]
That's correct, the RTC doesn't actually calculate the day of the week, it just increments it at midnight.
